itwin-platform-access/imodels-access-backend/package.json

```diff
@@ -18,10 +18,22 @@
     "name": "Bentley Systems, Inc.",
     "url": "http://www.bentley.com"
   },
+  "type": "module",
   "main": "lib/index.js",
-  "typings": "lib/index",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./lib/esm/index",
+        "default": "./lib/esm/index.js"
+      },
+      "require": {
+        "types": "./lib/cjs/index",
+        "default": "./lib/cjs/index.js"
+      }
+    }
+  },
   "scripts": {
-    "build": "tsc 1>&2",
+    "build": "tsc 1>&2 --outDir lib/esm",
```

Comment:
you only build for esm, we will need to build for cjs as well

Reply:
and we should probably do it for the other pkgs we publish in this monorepo as well